### DISCUSSION TITLE
Changes made elsewhere will not be synchronized to the local

### DISCUSSION THREAD

**Discussion by huguadao** [iterate-ch/cyberduck#16698] [Q&A]
The network disk mounted by webdav will not be actively synchronized to the local from other places, including the addition, deletion and modification of mountain duck on another computer, and will not be updated for long.

**Comment by dkocher** [comment]
Do the changes show if you select [Reload](https://docs.mountainduck.io/mountainduck/interface/#reload) from the context menu?

**Reply by huguadao** [reply]
By reload the command, you can synchronize to the latest changes, but shouldn't this be automatic?

**Reply by huguadao** [reply]
In addition, the folder for offline access has been set up. After restarting the computer, to access the folder, there will be a load chart. If the file has been cached locally, why is there a load chart?
